docs/hello_nextflow/02_hello_world.md

```diff
@@ -822,13 +833,13 @@ In most cases, when we run on multiple inputs, the input values are contained in
 
 Nextflow offers a variety of predefined operators and functions for reading data in from common file formats and applying text transformations. In this example, we used the [`fromPath()`](https://www.nextflow.io/docs/latest/channel.html#frompath) channel factory with the [`splitText()`](https://www.nextflow.io/docs/latest/operator.html#splittext) operator to read each line as a separate value, then we used a [closure](https://www.nextflow.io/docs/latest/script.html) to apply the [`trim()`](https://www.tutorialspoint.com/java/java_string_trim.htm) method to strip the newline (`\n`) character from each element.
 
-#### 1. Modify the channel declaration to take an input file (through a parameter) instead of hardcoded values
+#### 1. Modify the channel declaration to take an input file (through a parameter) instead of a single parameter
```

Comment:
We need to add an explanation here of why we're switching from `Channel.of()` to `Channel.fromPath()`

I would take out the paragraph above (which again shows our full hand in one big infodump before the learner is ready for it) and instead have a brief explanation after each subheader of what's happening at that specific step. 

(Here too lmk if you'd prefer I just push a commit with the changes)